spec.html

```diff
@@ -24800,6 +24800,7 @@ <h1>The Global Object</h1>
     <li>does not have a [[Construct]] internal method; it cannot be used as a constructor with the `new` operator.</li>
     <li>does not have a [[Call]] internal method; it cannot be invoked as a function.</li>
     <li>has a [[Prototype]] internal slot whose value is implementation-dependent.</li>
+    <li>is an instance of the standard built-in Object constructor.</li>
```

Comment:
Is “is an instance of the standard built-in Object constructor” an observable status / a defined concept? 

I’m guessing the intention here is to say that the [[Prototype]] chain (initially? permanently?) includes %Object.prototype%, which is not the same thing as being an instance of a given constructor as this phrase is used elsewhere. For example,

> [A Boolean Object is] a member of the Object type that is an instance of the standard built-in Boolean constructor.

It’s observable (indirectly) that an object is a Boolean Object because the [[BooleanData]] slot is obtained uniquely through creation with the Boolean constructor. It’s incidental that such an object likely has %Boolean.prototype% as its [[Prototype]], and it may not — `Object.setPrototypeOf(new Boolean, null)` is still a Boolean Object.

Reply:
I agree with your opinion. The sentence "is instance of the the standard built-in Object constructor" does not clearly imply that the prototype chain includes %Object.prototype%. Thus, I just directly wrote that the prototype chain include %Object.prototype% and updated the pull request.